course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- [ ] Chaos‐tribolium example, time series evidence
+- [ ] Non‐stationarity and critical transitions
+- [ ] Metapopulation dynamics
+- [ ] Source sink dynamics, synchrony, portfolio effects
+- [x] Dispersal, spatial spread, traveling waves
+- [ ] Spread of infectious diseases
+- [ ] Competition (phase portrait, case of coexistence)
+- [ ] Competition (other scenarios)
+- [x] Lotka‐voltera predator‐prey dynamics
+
+
+## The original RQM course
+I have written a short comment under each heading and then pasted from Christie's notes online to easier get an overview of what is included in the original course syllabus.
+
+### Lecture topics
+These are all the topics/bullet points from Christies course, in chronological order. I put a checkmark next to the topics I think are necessary to include, please feel free to add/remove checkmarks, just include comments with the rationale.
+
+- [x] Course syllabus and expectations
+- [x] Open science, open data, & reproducibility
+- [ ] Rules and regulations from funders and institutions
+- [ ] Overview of skills for collaborative science practice
+- [x] Using other people’s data - intro to datasets
+- [x] Spreadsheet best practices
+- [x] Introduction to metadata
+- [ ] Data creation and authorship
+- [x] Introduction to data cleaning
+- [ ] Grey data liberation
+- [x] Introduction to Scripting
+- [x] Data Cleaning in R
+- [x] Version Control
+- [x] R Review
+- [x] The basics of programming in R
+- [x] Even more with R (writing simple software)
+- [ ] Getting a bit abstract, submit to conference
+- [x] Further topics in R
+- [x] Visual communication with data
+- [ ] Ethics and risks in sharing data
+- [x] Plotting in R
+- [ ] Academic publishing, guest speaker
+- [x] Project progress via github task list
+- [x] Supported work period, Christie away
+- [ ] More on Citizen science approaches, Christie away
+- [ ] Special Guest: Auriel Fournier
+- [x] Supported work period
+- [ ] Open source beehives, Christie away
+- [x] Supported work period
+- [x] Supported work period
+- [ ] Special guest mozilla open science
+- [x] Preparing a paper for publication
+
+### Discussions
+The original RQM syllabus includes lectures, exercises and discussions. While the themes of the exercises largely map well to the lecture topics, the discussions are weighted more toward openness and publishing in general. Below is a list of discussion topics from the RQM course. We need to discuss how many of these we want to include, and if we think the time can be better spent on the lectures and tutorials (this will affect the grading also since some of the blog posts are to reflect on the discussion topics).
+- Openness and reproducibility in research
+    - So why bother with reproducibility? What's the big deal about openness? How do they fit together? in this, the first class discussion, let's think about your motivations for taking this class. We'll begin the discussion by watching the video together. 
```

Comment:
Yes, I think this should be part of the first class. Potentially expanded upon as discussions later on.